checkout-ui-custom/src/_js/_customAddressForm.js

```diff
@@ -729,13 +729,19 @@ class fnsCustomAddressForm {
 
     const _this = this
     const country = _countries.find(c => c[1] === countryCode)
+    const state = shippingData.address.state
```

Comment:
if `shippingData.address` is `undefined` then it will breaking the app when calling state

Reply:
and when i save some address this `shippingData` instance is outdated, you need to get again the order form in this point to get the current state.

You could use in this line `shippingData.selectedAddresses()` to get the last selected address.

Reply:
I changed it to read the address form object that maintains the reference to the address